Harvest.Core/Harvest.Core.csproj

```diff
@@ -14,6 +14,7 @@
     <PackageReference Include="Microsoft.EntityFrameworkCore.Relational" Version="5.0.1" />
     <PackageReference Include="Microsoft.EntityFrameworkCore.SqlServer" Version="5.0.1" />
     <PackageReference Include="NetTopologySuite" Version="2.1.0" />
```

Comment:
@sprucely Does NetTopologySuite need the latest stable (2.20) for the one you added below?

Reply:
It probably wouldn't hurt. But I think I'll do it in another PR when we're updating a bunch of dependencies.